### ISSUE TITLE
Compiler assertion when using EXT_buffer_reference

### ISSUE THREAD

**Issue by godlikepanos** [GPUOpen-Tools/radeon_gpu_analyzer#82]
Hi,

I'm getting the following assertion: 

```
Error: internal assert: Bil::BilInstructionConvert::ConvertToIl(198) : pDestType->IsAggregate() == false.
``` 

when compiling the following GLSL shader:

```
#version 460 core

#extension GL_EXT_buffer_reference : enable
#extension GL_EXT_buffer_reference2 : enable
#extension GL_EXT_shader_explicit_arithmetic_types_int64 : enable
#extension GL_EXT_nonuniform_qualifier : enable
#extension GL_EXT_scalar_block_layout : enable

struct PointLight2
{
	vec3 m_diffuseColor;
};

layout(buffer_reference, scalar, buffer_reference_align = 4) buffer PointLight2Ref
{
	PointLight2 m_value;
};

struct ClustererUniforms
{
    uint64_t m_pointLightAddress;
};

layout(binding = 0) buffer b
{
    ClustererUniforms u_unis;
};

layout(location = 0) out vec3 fragColor;

void main()
{
    fragColor = PointLight2Ref(u_unis.m_pointLightAddress).m_value.m_diffuseColor;
}
```

Please not that the system that runs RGA doesn't have an AMD GPU and from what I understand RGA will default to `amdvlk64.dll`.

**Comment by AmitBM** [comment]
Hi godlikepanos,

Thank you for reporting this issue.

A fix will be provided in the next RGA release.

**Comment by AmitBM** [comment]
Hi godlikepanos,

This should be resolved in the latest release (RGA 2.5).
